src/utils/width.css

```diff
@@ -0,0 +1,107 @@
+.u-w-full {
```

Comment:
This should be `.u-w-12/12` to keep it consistent with the grid classes.

Reply:
Well, you also have 6/6 etc. So thats why I went with `full`.

Reply:
Both would, and should, do the same thing.

Reply:
:+1: makes sense!